easybuild/tools/toolchain/toolchain.py

```diff
@@ -331,8 +331,7 @@ def show_variables(self, offset='', sep='\n', verbose=False):
         if self.vars is None:
             self.generate_vars()
 
-        var_names = self.variables.keys()
-        var_names.sort()
+        var_names = sorted(self.variables.keys())
         res = []
         for v in var_names:
```

Comment:
Do you mind getting rid of this single-letter variable here while you're at it?

Those shouldn't be used outside of list comprehensions...

```python
for var_name in var_names:
```

or

```python
for key in var_names:
```

or even

```python
for key in sorted(self.variables.keys()):

Reply:
Unrelated